_posts/en/newsletters/2022-05-18-newsletter.md

```diff
@@ -0,0 +1,579 @@
+
+  - url: https://twitter.com/nvk/status/1455681554047545352
+    author: Rodolfo Novak
+    title: "CEO and Co-Founder of Coinkite"
+    text: |
+      Follow @bitcoinoptech and sign up for their news letter, 5-9 signal.
+
+      Very few sources of bitcoin technical information are this well summarized and explained.
+
+      Do it now, so that you don’t forget 😉
+
+  - url: https://twitter.com/Kristian_Kho/status/1452201573799632896
+    author: Kristian Kho
+    text: |
+      There’s plenty of newsletters around crypto stuff to keep you up to date.
+
+      One of the ones I’ve religiously followed has been @bitcoinoptech.
+
+      highly recommended if you hold some btc and wants to see real work being done to make the protocol better.
+
+  - url: https://twitter.com/ElleMouton/status/1418108253096095745
+    author: Elle Mouton
+    title: Engineer, Lightning Labs
+    text: |
+      A bit behind on Taproot? There is still time and luckily for you
+      @bitcoinoptech has a GREAT workshop that will bring you up to
+      speed: https://github.com/bitcoinops/taproot-workshop/
+
+      Can highly recommend. It goes through all the building blocks
+      you need. Go do iiiiiiiit 💪 #Bitcoin #taproot
+
+  - url: https://twitter.com/Empact/status/1403702689737871362
+    author: Ben Woosley
+    title: Bitcoin Core Developer and Senior Developer at Unchained Capital
+    text: "@bitcoinoptech does a good job explaining the tech to a technical integrators audience, I would watch their workshop"
+
+  - url: https://twitter.com/RajarshiMaitra/status/1398931136714182656
+    author: Rajarshi Maitra
+    text: "[I]t's almost a full time job trying to summarise what's happening everywhere.  That's what guys at @bitcoinoptech does, so just following their news letters will give good enough understandings."
+
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **When would enabling `OP_CAT` allow recursive covenants?** in a
+  discussion on the Bitcoin-Dev mailing list about re-adding the
+  [OP_CAT][op_cat] opcode removed from Bitcoin in 2010, developer
+  ZmnSCPxj [claimed][zmnscpxj cat1] `OP_CAT` needed to be combined with
+  proposed opcodes `OP_TX` (see [Newsletter #187][news187 op_tx]),
+  [OP_CHECKSIGFROMSTACK][topic op_checksigfromstack] (CSFS), or a
+  similar feature in order for it to enable inescapable recursive
+  [covenants][topic covenants]---contracts which use Bitcoin's
+  consensus rules to enforce that all bitcoins received to the contract
+  can only be spent to the same contract. 
+
+    Recursive covenants depend on a technique referred to as
+    *transaction introspection* by which a opcode can analyze part of
+    the transaction which executes that opcode.  Existing opcodes
+    `OP_CHECKSIG`, [BIP65][] `OP_CHECKLOCKTIMEVERIFY`, and [BIP112][]
+    `OP_CHECKSEQUENCEVERIFY` provide limited introspection.  Proposed
+    opcodes TX and CSFS could allow flexible introspection of all parts
```

Comment:
I agree that this would be clearer, but a standard piece of typographical advice is to try to minimize the use of special elements like bold, italics, background colors, or alternative fonts.  To that end, I usually try to avoid repetition of opcode names which would require use to put them in typewriter text.

Reply:
Yeah, I think my instinct was mostly prompted on "TX" being such a common abbreviation for transaction in general, that it didn't register as an opcode name to me.